sstables/sstables.hh

```diff
@@ -1184,6 +1184,9 @@ public:
 
 std::unique_ptr<sstable_stream_sink> create_stream_sink(schema_ptr, sstables_manager&, const data_dictionary::storage_options&, sstable_state, std::string_view component_filename, bool last_component);
 
+std::tuple<entry_descriptor, sstring, sstring> parse_path(const std::filesystem::path& sst_path);
+entry_descriptor parse_path(const std::filesystem::path& sst_path, sstring ks, sstring cf);
```

Comment:
Looks like you copied these declarations from `sstables/open_info.hh` now it is declared in two places?

Reply:
no, I wrote them myself.
I'll look into the double declaration.  don't remember seeing it...

Reply:
Then where is the definition of these functions?
I'm talking about [these](https://github.com/scylladb/scylladb/blob/f8800b3f19d7063a678c72c6b46bd90da3e0c9cf/sstables/open_info.hh#L50) declarations
Something is tremendously confusing me here

Reply:
Ah, I think I know what happened, you saw the function in the `cc` file, but not in the header, so you added the declaration to `sstables.hh` thinking these were static functions in the compilation unit without a declaration, but in fact it was already declared in `open_info.hh`

Reply:
Right

Reply:
reused the existing declarations in v2 ([06da3f3](https://github.com/scylladb/scylladb/commit/06da3f30f0c2da5e684fbff7b517e97d70d70559))
and using the 2nd variant which is simpler